src/instruments/src/EFB/Ground/Ground.scss

```diff
@@ -59,49 +59,62 @@
 
 .control-grid > div {
   @apply rounded-lg;
```

Comment:
We should try to move this all to in-html TW in the future. Not necessary for now